### PR TITLE
CMR-9694: Updates the README

### PR DESCRIPTION
**Pull request by abbottry**
Following the README to setup CMR was confusing, I went through the process and made any relevant updates as well as removed unnecessary steps that were causing confusion.

### CODE REVIEW

**Review by Bccorb — APPROVED** [review]

**Review by macrouch — APPROVED** [review]

**Review by zimzoom — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -82,45 +63,53 @@ echo "export PATH=\$PATH:`pwd`/bin" >> ~/.profile
...
 
-1. `cmr setup profile` and then update the new `./dev-system/profiles.clj` file.
-   it will look something like this:
    ``` clojure
-   {:dev-config {:env {:cmr-metadata-db-password "<YOUR PASSWORD HERE>"
-                       :cmr-sys-dba-password "<YOUR PASSWORD HERE>"
-                       :cmr-bootstrap-password "<YOUR PASSWORD HERE>"
-                       :cmr-ingest-password "<YOUR PASSWORD HERE>"
-                       :cmr-urs-password "<YOUR PASSWORD HERE>"}}}
+   {:dev-config {:env {:cmr-metadata-db-password "CHANGE_ME"
```
Comment (jceaser) [inline README.md:75]: did you mean to make this change?

File: README.md [hunk 2]
```diff
@@ -132,53 +121,11 @@ after entering `uberjar` in each step above.
...
-done
-```
-
-#### Dependency Ambiguities and `.jar` File Conflicts
-
-To see if the JVM is having problems resolving which version of a
-dependency to use, you can run `cmr test dep-tree PROJ`. To perform this
-against all projects: `cmr test dep-trees`.
+## Using VS Code Clava extenstion
+VS Code has an extension that allows for easier develop of clojure applications.
```
Comment (jceaser) [inline README.md:125]: please consider putting satirical emoji quotes around the word easier

File: README.md [hunk 3]
```diff
@@ -300,124 +247,65 @@ with the standard being `:unit` and `:integration`.
...
-  - The UMM Spec lib contains JSON schemas that define the Unified Metadata
-    Model, as well as mappings to other supported formats, and code to migrate
-    collections between any supported formats.
-
-## Further Reading
-
-- [CMR Client Partner User Guide](https://wiki.earthdata.nasa.gov/display/ED/CMR+Client+Partner+User+Guide)
-- [CMR Data Partner User Guide](https://wiki.earthdata.nasa.gov/display/CMR/CMR+Data+Partner+User+Guide)
-
-## License
```
Comment (jceaser) [inline README.md:421]: should we not continue to leave this at the bottom like everything else on the web?

**Review by jceaser — CHANGES_REQUESTED** [review]
I think at the very least, bump up the year to 2024

File: README.md [hunk 4]
```diff
@@ -308,4 +308,18 @@ applications, as well as several libraries and support applications.
    instance of ECHO. It will only provide the parts necessary to enable
    integration testing. You should not expect a perfect or complete
    implementation of ECHO.
-  - Main method: cmr.mock_echo.runner
\ No newline at end of file
+  - Main method: cmr.mock_echo.runner
+  
+# License
```
Comment (jceaser) [inline README.md:313]: should be an h2, only one h1 should be in the document. Also, this is the second License section

**Review by jceaser — APPROVED** [review]

**Pull request merged**